source/generate.py

```diff
@@ -39,7 +42,7 @@ def __init__(
                 f"Sequence length set to {self.max_length} BASED ON 'model.config.n_positions'"
             )
 
-        self.temperature = temperature
+        self.temperature = TEMPERATURE
```

Comment:
it wouldn't change anything but this probably should be lower letters to refer the what is passed

Reply:
done